packages/react/src/components/SignIn/__tests__/__snapshots__/SignIn.test.tsx.snap

```diff
@@ -26,14 +26,20 @@ exports[`SignIn should match the snapshot 1`] = `
             class="oxygen-text-field"
           >
             <label
+<<<<<<< HEAD
```

Comment:
This is a merge conflict right?
If so, Remove the snapshot and reupdate.

Reply:
Fixed with https://github.com/wso2/oxygen-ui/pull/38/commits/f2ee9ffa587d3b161c78343d67540bd081438984